.bazelrc

```diff
@@ -25,7 +25,7 @@ common --noincompatible_disallow_empty_glob
 
 # Don't try and auto detect the cc toolchain, as we use our own gcc toolchains.
 common --action_env=BAZEL_DO_NOT_DETECT_CPP_TOOLCHAIN=1
-common --incompatible_enable_cc_toolchain_resolution
+common --incompatible_enable_cc_toolchain_resolution --incompatible_enable_proto_toolchain_resolution
```

Comment:
🧙

Reply:
I heard about it on a podcast last week!